### PR TITLE
chore: remove language switching code

### PR DESCRIPTION
**Pull request by mvadari**
## High Level Overview of Change

There is a bunch of code in `Footer` that isn't doing anything, regarding switching languages. This PR removes that code.

### Context of Change

The code isn't used anyways, since we don't have very many translations. The design also doesn't really fit the rest of our site, so it'd need to be redesigned anyways.

There are some Vite issues regarding this code, so it seems easier to just get rid of it.

### Type of Change

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Before / After

No visible changes.

## Test Plan

Works locally.

### CODE REVIEW

File: src/containers/Footer/index.jsx [hunk 1]
```diff
@@ -229,7 +114,6 @@ const Footer = (props) => {
 }
 
 Footer.propTypes = {
-  language: PropTypes.string.isRequired,
   actions: PropTypes.shape({
     updateLanguage: PropTypes.func,
```
Comment (ckniffen) [inline src/containers/Footer/index.jsx:118]: You can remove updateLanguage too which will allow the removal of redux and props from this component completely

File: src/containers/Footer/index.jsx [hunk 2]
```diff
@@ -1,130 +1,16 @@
...
-import classnames from 'classnames'
-
-import { analytics, ANALYTIC_TYPES } from '../shared/utils'
 import { updateLanguage } from '../App/actions'
 
 import packageConfig from '../../../package.json'
 import { ReactComponent as Logo } from '../shared/images/XRPLedger.svg'
-import iconDownArrow from '../shared/images/down_arrow.svg'
-import iconCheck from '../shared/images/checkmark_small_green.png'
-import flagUSA from '../shared/images/flag_usa.png'
```
Comment (ckniffen) [inline src/containers/Footer/index.jsx:15]: Should we just delete all these images too and maybe translation files that are never updated?

**Review by ckniffen — CHANGES_REQUESTED** [review]
`updateLanguage` in /App/actions can be eliminated too.

File: src/containers/Footer/index.jsx (hunk 2 above)
Comment (mvadari) [reply src/containers/Footer/index.jsx:15]: I'm down to remove the images, but I think the translation files should stay - if/when we do decide to support multiple languages, it'll be a useful starting point.

**Review by ckniffen — APPROVED** [review]

**Review by jonathanlei — APPROVED** [review]

**Pull request merged**